scripts/dashboard-importer/import.sh

```diff
@@ -14,36 +14,34 @@
 # limitations under the License.
 
 JSON_PATH=$1 # Path to directory containing JSON files or a JSON file
-PROJECT=$2 # Supplied GCP project to upload to
+PROJECT=$2 # Optionally supplied GCP project to upload to
 
 # Validate Arguments
 if [ "$JSON_PATH" = "" ]
 then
   echo "Error: No JSON_PATH included"
-  echo "Syntax: import.sh <JSON_PATH> <PROJECT>"
-  exit
-fi
-
-if [ "$PROJECT" = "" ]
-then
-  echo "Error: No PROJECT included"
-  echo "Syntax: import.sh <JSON_PATH> <PROJECT>"
+  echo "Syntax: ./import.sh <JSON_PATH> <PROJECT>"
   exit
 fi
 
 # Convert dashboards
 CONVERSION_OUTPUT=$(node dist/convert.js dashboards $JSON_PATH)
 echo "$CONVERSION_OUTPUT"
-echo
-echo "Conversion Complete. Proceeding to uploading..."
 
-# Extract output directory from console output
-DIRECTORY=$(grep -Po '(reports.*/)' <<< "$CONVERSION_OUTPUT" | sed -n '2p')
-echo
-echo -e "Now running: \033[34m./upload.sh $DIRECTORY $PROJECT\033[0m\n"
+# If project provided then upload the converted daashboard to the project
+if [ -n "$2" ]
+then
+  echo
+  echo "Conversion Complete. Proceeding to uploading..."
```

Comment:
Let's be consistent with title case vs sentence case. Also, "Proceeding to upload..."